torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1078,6 +1078,30 @@ def sample_inputs_max_min_reduction_no_dim(op_info, device, dtype, requires_grad
                                           requires_grad=requires_grad),))
     return inputs
 
+def sample_inputs_topk(op_info, device, dtype, requires_grad, **kwargs):
+    def get_tensor_input(size):
+        return make_tensor(size, device, dtype, requires_grad=requires_grad)
+
+    inputs = []
+    inputs.append(SampleInput(get_tensor_input((S, M, S)), args=(3,)))
+    inputs.append(SampleInput(get_tensor_input((S, M, S)), args=(3, 1)))
+    inputs.append(SampleInput(get_tensor_input((S, M, S)), args=(3, -2)))
```

Comment:
Nice new case

Reply:
These are not new.
The old spec was specifying which args were dimension arguments and were automatically testing `val` and `val - size()` to make sure we do the wrapping properly.
I didn't find an arg in OpInfo to specify that so I added the tests manually.

Reply:
Cool!